wgsl/index.bs

```diff
@@ -12642,6 +12838,40 @@ the rules in [[#floating-point-rounding-and-overflow]] apply.
   <tr><td>`unpack2x16snorm(x)`<td>3 ULP<td>N/A
   <tr><td>`unpack2x16unorm(x)`<td>3 ULP<td>N/A
   <tr><td>`unpack2x16float(x)`<td>Correctly rounded<td>N/A
+  <tr><td>`subgroupBroadcast(x, i)`<td colspan=2>Correctly rounded
+  <tr><td>`subgroupBroadcastFirst(x)`<td colspan=2>Correctly rounded
+  <tr><td>`subgroupAdd(x)`<td colspan=2>Inherited from &sum;x for all [=subgroups/active=] invocations in the subgroup
+
+      Note: The order of operations is implementation dependent.
```

Comment:
It's a bit cruel to say that the precision is inherited from an operation, `&sum;` that is nowhere defined. The order in which the individual additions are associated is essential, so it shouldn't be in a `Note`. Suggest:

    Inherited from &sum;x for all [=subgroups/active=] invocations in the subgroup, where the associativity of the individual additions is implementation dependent.

Reply:
Saying "inherited from" already allows mathematically pure alternatives so the order of operations is not specified. I'll try to clean these up though.